autogalaxy/operate/deflections.py

```diff
@@ -361,7 +361,7 @@ def tangential_critical_curve_list_from(
         tangential_eigen_values = self.tangential_eigen_value_from(grid=grid)
 
         tangential_critical_curve_indices_list = measure.find_contours(
-            tangential_eigen_values.native, 0
+            np.array(tangential_eigen_values.native), 0

```

Comment:
This is necessary in a lot of places to ensure that a numpy array instead of an autoarray array is passed to numpy functions. If the object is already a numpy array it has no effect; if it is an autoarray array the underlying numpy array is returneed